src/prefect/tasks/monte_carlo/monte_carlo_lineage.py

```diff
@@ -0,0 +1,468 @@
+        - destination (dict, optional): a destination node configuration. Expected to include
+        the following keys: `node_name`, `object_id`, `object_type`, `resource_name`,
+        and optionally also `metadata_key` and `metadata_value`.
+        - api_key_id (string, optional): to use this task,
+        you need to pass the Monte Carlo API credentials.
+        Those can be created using https://getmontecarlo.com/settings/api.
+        To avoid passing the credentials
+        in plain text, you can leverage the PrefectSecret task in your flow.
+        - api_token (string, optional): the API token.
+         To avoid passing the credentials in plain text,
+         you can leverage PrefectSecret task in your flow.
+        - prefect_context_tag (bool, optional): whether to automatically add
+        a tag with Prefect context.
+        - expire_at (string, optional): date and time indicating when to expire
+        a source-destination edge. You can expire specific lineage nodes.
+        If this value is set, the
+        edge between a source and destination nodes will expire on a given date.
+        Expected format: "YYYY-MM-DDTHH:mm:ss.SSS". For example, "2042-01-01T00:00:00.000".
+        - **kwargs (dict, optional): additional keyword arguments to pass
+        to the Task constructor.
+
+        Example source:
+
+        ```python
+        source = dict(
+            # this may be a shorter version of object_id
+            node_name="example_table_name",
+            object_id="example_table_name",
+            # "table" is recommended, but you can use any string, e.g. "csv_file"
+            object_type="table",
+            resource_name="name_your_source_system",
+            metadata_key="your_key_name",
+            metadata_value="your_abitrary_value",
+        )
+        ```
+
+        Example destination:
+
+        ```python
+        destination = dict(
+            # the full name of a data warehouse table
+            node_name="db_name:schema_name.table_name",
+            object_id="db_name:schema_name.table_name",
+            # "table" is recommended, but you can use any string, e.g. "csv_file"
+            object_type="table",
+            resource_name="your_dwh_resource_name",
+            metadata_key="your_optional_key_name",
+            metadata_value="your_optional_abitrary_value",
+        )
+        ```
+
+    """
+
+    def __init__(
+        self,
+        source: Dict[str, str] = None,
+        destination: Dict[str, str] = None,
+        api_key_id: str = None,
+        api_token: str = None,
+        prefect_context_tag: bool = True,
+        expire_at: str = None,
+        **kwargs: Any,
+    ):
+        self.source = source
+        self.destination = destination
+        self.api_key_id = api_key_id
+        self.api_token = api_token
+        self.prefect_context_tag = prefect_context_tag
+        self.expire_at = expire_at
+        self._catalog_url = "https://getmontecarlo.com/catalog"
```

Comment:
If `_catalog_url` cannot be set by the user, I think it makes sense for it to be either a class variable or variable declared in this module, but external to the class.